Documentation/common-problems.md

```diff
@@ -23,25 +26,149 @@ One of the first things that should be done is to start the [rook-tools pod](./t
+```
+If you encounter this, just **restart the Kubelet process**, as described in the **[Restart Kubelet](./kubernetes.md#deploy-rook)** section of the Rook deployment guide.
+
+### Volume Creation
+The volume must first be created in the Rook cluster and then bound to a volume claim before it can be mounted to a pod.
+Let's confirm that with the following commands and their output:
+```console
+> kubectl get pv
+NAME                                       CAPACITY   ACCESSMODES   RECLAIMPOLICY   STATUS     CLAIM                    STORAGECLASS   REASON    AGE
+pvc-9f273fbc-bdbf-11e7-bc4c-001c428b9fc8   20Gi       RWO           Delete          Bound      default/mysql-pv-claim   rook-block               25m
+
+> kubectl get pvc
+NAME             STATUS    VOLUME                                     CAPACITY   ACCESSMODES   STORAGECLASS   AGE
+mysql-pv-claim   Bound     pvc-9f273fbc-bdbf-11e7-bc4c-001c428b9fc8   20Gi       RWO           rook-block     25m
+```
+Both your volume and its claim should be in the `Bound` status.
+If one or neither of them is not in the `Bound` status, then look for details of the issue in the `rook-operator` logs:
+```console
+kubectl -n rook-system logs `kubectl -n rook-system -l app=rook-operator get pods -o jsonpath='{.items[*].metadata.name}'`
+```
+If the volume is failing to be created, there should be details in the log output, especially those tagged with `op-provisioner`.
+
+### Volume Mounting
+The final step in preparing Rook storage for your pod is for the `rook-agent` pod to mount and format it.
+If all the preceding sections have been successful or inconclusive, then take a look at the `rook-agent` pod logs for further clues.
+You can determine which `rook-agent` is running on the same node that your pod is scheduled on by using the `-o wide` output, then you can get the logs for that `rook-agent` pod similar to the example below:
+```console
+> kubectl -n rook-system get pod -o wide
+NAME                             READY     STATUS    RESTARTS   AGE       IP             NODE
+rook-agent-h6scx                 1/1       Running   0          9m        172.17.8.102   172.17.8.102
+rook-agent-mp7tn                 1/1       Running   0          9m        172.17.8.101   172.17.8.101
+rook-operator-2203999069-3tb68   1/1       Running   0          9m        10.32.0.7      172.17.8.101
+
+> kubectl -n rook-system logs rook-agent-h6scx
+2017-10-30 23:07:06.984108 I | rook: starting Rook v0.5.0-241.g48ce6de.dirty with arguments '/usr/local/bin/rook agent'
+...
+```
+
+In the `rook-agent` pod logs, you may see a snippet similar to the following:
+```console
+Failed to complete rbd: signal: interrupt.
+```
+In this case, the agent waited for the `rbd` command to finish but it did not in a timely manner so it gave up.
```

Comment:
typo?: "it did not <respond> in a timely manner"

Reply:
i intended it to mean that the agent was waiting for `rbd` to finish, but it (rbd) did not finish.  does this read better?
> In this case, the agent waited for the `rbd` command but it did not finish in a timely manner so it gave up.